### ISSUE TITLE
app crashed. [NRMAHTTPUtilities startTrip] EXC_BAD_ACCESS (KERN_INVALID_ADDRESS)

### ISSUE THREAD

**Issue by hohoins** [newrelic/newrelic-flutter-agent#126]
The app suddenly crashed while running.
After analyzing the exception, it turned out that the crash occurred inside the NewRelic flutter SDK.
Please fix this because it is a critical issue.
No special reproduction method has been confirmed.
The frequency of reproduction is not high. It occurs intermittently.

***
### Log
```
com.apple.root.default-qos
EXC_BAD_ACCESS KERN_INVALID_ADDRESS 0x00005dac8caa4880
Crashed: com.apple.root.default-qos
0  libobjc.A.dylib                0x4820 objc_msgSend + 32
1  NewRelic                       0x74f74 +[NRMAHTTPUtilities startTrip] + 195 (NRMAHTTPUtilities.mm:195)
2  NewRelic                       0x5bb58 __118+[NRMANetworkFacade noticeNetworkRequest:response:withTimer:bytesSent:bytesReceived:responseData:traceHeaders:params:]_block_invoke + 155 (NRMANetworkFacade.mm:155)
3  libdispatch.dylib              0x26a8 <redacted> + 32
4  libdispatch.dylib              0x4300 <redacted> + 20
5  libdispatch.dylib              0x744c <redacted> + 928
6  libdispatch.dylib              0x15be4 <redacted> + 392
7  libdispatch.dylib              0x163ec <redacted> + 156
8  libsystem_pthread.dylib        0x1928 _pthread_wqthread + 228
9  libsystem_pthread.dylib        0x1a04 start_wqthread + 8

```

***
### Environment
newrelic_mobile: 1.1.1
flutter: 3.13.9

**Comment by ndesai-newrelic** [comment]
@hohoins The flutter agent 1.1.6 is now released, and the issue reported in #126 should be fixed. Please let us know if you experience this issue again.

**Comment by hohoins** [comment]
@ndesai-newrelic 
👍👍👍👍👍 
Thank you for fixing the issue.
I applied flutter agent 1.1.6.
If the problem occurs again, I will report it.
Thank you.
